boards/arm/particle_xenon/Kconfig

```diff
@@ -0,0 +1,14 @@
+# Kconfig - Particle Xenon configuration
+#
+# Copyright (c) 2018 Endre Karlson
+#
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_PARTICLE_XENON
+
+config BOARD_ENABLE_DCDC
```

Comment:
This depends on CONFIG_BOARD_PARTICLE_XENON, i.e. clearly depends on a particular board, while introduces an option generally named CONFIG_BOARD_ENABLE_DCDC . This is highly confusing.

Nor I personally think that such an option (even named non-confusingly is needed), given that CONFIG_SOC_DCDC_NRF52X already exists. Just use it, and make your board code do things depending on whether it's set or not.

And if you're concerned how users would discover that option - well, that's what docs are for ;-).

Reply:
Ok, removing my -1, but I the argument that it's confusing still holds ;-).

Reply:
Confusing and doesn't scale - we can't duplicate each  Zephyr option thru board-specific alias, so there should be no reasons to do that even for some options.